test/views/CalculationViewSpec.scala

```diff
@@ -48,18 +48,21 @@ class CalculationViewSpec extends SpecBase with ViewBehaviours with ViewUtils {
     val fifteenDayCalculation = "#15-day-calculation"
 
     val thirtyDayCalculation = "#30-day-calculation"
+
+    val whenPenaltyIncreases = "#when-it-increases"
+
+    val calculation = "#calculation"
   }
 
   "CalculationView" should {
 
     "if it is an additional penalty and the penalty is estimated" must {
```

Comment:
So in the above test description we have"if it is an additional penalty and the penalty is estimated" can this be "if it is second late payment penalty and the penalty is estimated"